src/pages/App.js

```diff
@@ -7,78 +7,114 @@ import Navbar from './components/Navbar';
 import Demo from './components/Demo';
 import Card from './components/Card';
 import Footer from './components/Footer';
+import Statistics from './components/Statistics/Statistics';
 
 const App = ({ searchWord, words }) => {
   const router = useRouter();
   return (
     <div className="overflow-x-hidden" id="homepage-container">
       <Navbar isHomepage />
       <div>
-        <div className="flex flex-col justify-center w-screen h-screen p-0 pt-10 lg:pt-32">
+        <div className="flex flex-col justify-center w-screen h-screen p-0 pt-10 lg:pt-16">
           <FadeIn>
-            <h1 className="text-4xl md:text-6xl font-extrabold text-center mt-24 lg:mt-6 mb-24">
-              The First African Language API
+            <h1 className="text-4xl md:text-6xl ml-16 font-extrabold mt-24">The First African</h1>
+            <h1 className="text-4xl md:text-6xl ml-20 font-extrabold">
+              Language
+              <span className="text-green-500">API</span>
             </h1>
           </FadeIn>
-          <div className="text-xl md:text-2xl text-center text-gray-700 w-9/12 md:w-7/12 self-center mb-24">
+          <div className="text-xl md:text-1xl w-9/12 md:w-7/12 ml-20 mb-4 mt-8 leading-10">
```

Comment:
I don't think we need `md:text-1xl` rather `md:text-xl` should be good!

Reply:
Yes, i have fixed it.